code/modules/admin/IsBanned.dm

```diff
@@ -247,6 +250,13 @@
 		. = list("reason" = "Stickyban", "desc" = desc)
 		log_suspicious_login("Failed Login: [ckey] [computer_id] [address] - StickyBanned [ban["message"]] Target Username: [bannedckey] Placed by [ban["admin"]]")
 
+	var/static/list/ckey_to_tz = list(
+		"fikou" = 2 HOURS
+	)
+
+	if(ckey in ckey_to_tz && timeofday >= ckey_to_tz[ckey] && timeofday <= MIN_TIME + ckey_to_tz[ckey])
```

Comment:
you want `REALTIMEOFDAY` I think, tho the ckey_to_tz is an offset which is strange. code's bad

Reply:
it's  late